styles/main.css

```
@@ -0,0 +1,7 @@
+img {
+    height: 30px;
+}
+
+.titleError, .authorError, emailError, .phoneError, .isbnError {
```

```suggestion
.hideError {
```
A single class can be used to hide all errors. When you need to show the error, remove the class from the element.